ci/checksizes.sh

```diff
@@ -34,21 +34,21 @@ REVERSI_OPT_GZ_SIZE=$(stat '-c%s' "$REVERSI_OPT.gz")
 
 case $FULLVER in
   2.11.12)
-    REVERSI_PREOPT_EXPECTEDSIZE=510000
+    REVERSI_PREOPT_EXPECTEDSIZE=544000
     REVERSI_OPT_EXPECTEDSIZE=112000
-    REVERSI_PREOPT_GZ_EXPECTEDSIZE=68000
-    REVERSI_OPT_GZ_EXPECTEDSIZE=29000
-    ;;
-  2.12.8)
-    REVERSI_PREOPT_EXPECTEDSIZE=595000
-    REVERSI_OPT_EXPECTEDSIZE=131000
     REVERSI_PREOPT_GZ_EXPECTEDSIZE=70000
     REVERSI_OPT_GZ_EXPECTEDSIZE=30000
```

Comment:
I am not entirely sure, but I think it could be because field names are more different across unrelated class hierarchies, so there are more distinct names, and that means GCC needs to use more distinct short names, which in turn can cause it to use longer names.

Reply:
Ah, fair enough. That makes sense.